### PR TITLE
Changed 'Wip' finding regex expression

### PR DESCRIPTION
**Pull request by Fogapod**

### CODE REVIEW

**Comment by Fogapod** [comment]
Just realized that my pull would be recognized as not valid

File: updater/updater.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ class Updater {
 	Checks if the PR should be merged
 
 	Return true if there amount of approved reviews is more than double the amount of unapproved reviews
-	and there are more than two approved reviews
+	and there are more than three approved reviews
```
Comment (matmen) [inline updater/updater.js]: this actually needs to be two

File: updater/updater.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ class Updater {
 		let changed = false;
 
 		for (const pullRequest of pullRequests.data) {
-			if (/^(\[|\()?WIP:?(\]|\))?|(\[|\()WIP(\]|\))$/i.test(pullRequest.title)) continue; // Don't check the PR if it's WIP
+			if (/(^|\W)WIP(\W|$)/i.test(pullRequest.title)) continue; // Don't check the PR if it's WIP
```
Comment (matmen) [inline updater/updater.js]: idk i dont think that this is better

**Comment by Fogapod** [comment]
So now 'WIP' will be ignored if in quotes or near characters "wipText"

File: updater/updater.js (hunk 1 above)
Comment (Fogapod) [reply updater/updater.js]: So need to change to >= 2 [there](https://github.com/matmen/NotSoChaos/pull/6/files/d10698ba89697a1e56fd73d45d370656e8006253#diff-72c2d776333a4a8abad6554a55f7a276L109)?

**Pull request closed without merging**